### PR TITLE
Show presence status for 1:1 chats

### PR DESCRIPTION
**Pull request by Kandr0s**
Inspired By #589 
Adds a small presence status bar next to the Contact picture.

screenshot

### CODE REVIEW

**Comment by uchchishta** [comment]
Hi, i don't know what iNPUTmice would say, but:
There's a trend currently in design I suppose, where people give color shadows for things, it looks like a glow. So, would you like to do something like that? And perhaps  change the glow to the buttom?

It would be nice to see statuses under avatars in private mucs, maybe.

**Comment by Kandr0s** [comment]
So, @iNPUTmice what do you think about such a feature.
Didn't want to put too much effort into it, if you don't want anything like that anyway.
If you are interested, i would keep working on it. maybe enable/disable it in settings, change the look of the status (maybe something like a small dogear, a bubble in one corner or any other suggestion)

**Comment by uchchishta** [comment]
@iNPUTmice can you comment on this?

**Comment by uchchishta** [comment]
@Kandr0s you might want to read older issues like #1937 and [faq](https://github.com/siacs/Conversations#where-can-i-see-the-status-of-my-contacts-how-can-i-set-a-status-or-priority)
> It's a design decision not to focus on presence.

**Comment by Kandr0s** [comment]
Thats why i said, i would implement it with a settings checkbox to show/hide this.

**Comment by participante0** [comment]
@Kandr0s, I like the idea, but unfortunately (from the FAQ):
> Making these status and priority optional isn't a solution either because Conversations is trying to get rid of old behaviours and set an example for other clients.

**Comment by Kandr0s** [comment]
Still, by default there are different presence status.
Online (green)
Display off (orange)
Vibratemode (orange)
Offline (red)

File: src/main/res/layout/conversation_list_row.xml [hunk 1]
```diff
@@ -28,7 +28,14 @@
                 android:layout_height="56dp"
                 android:layout_alignParentLeft="true"
                 android:scaleType="centerCrop"
-                app:riv_corner_radius="2dp"/>
+                app:riv_corner_radius_bottom_right="2dp"
+                app:riv_corner_radius_top_right="2dp"/>
```
Comment (Fank) [inline src/main/res/layout/conversation_list_row.xml]: You should not use corner radius, its against the material style.

Comment (Kandr0s) [inline src/main/res/layout/conversation_list_row.xml]: Well, tell @iNPUTmice and not me.

**Comment by whitetornado** [comment]
maybe a predefine status or icon for users to inform their contacts, like
<Difficult to reach> or a icon {microphone slash}
<in plane> or icon {plane}
<on my way> {road}
<answer later> {phone volume}

**Comment by licaon-kter** [comment]
@Kandr0s Still interested? Fix the conflicts

**Comment by iNPUTmice** [comment]
I’m actually not happy with how that looks like. I future version might copy Dinos grayed out style.

**Pull request closed without merging**